### PR TITLE
Adding chemcoord

### PR DESCRIPTION
**Pull request by ghutchis**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [X] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [X] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by ghutchis** [comment]
Willing to be listed as maintainer.

**Comment by ghutchis** [comment]
@conda-forge/help-python ready for review!

**Comment by ghutchis** [comment]
@conda-forge-admin, please ping @conda-forge/help-python

**Comment by mcocdawc** [comment]
Thank you and I confirm

File: recipes/chemcoord/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  patches:
+    - readme.patch
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
```
Comment (bernt-matthias) [inline recipes/chemcoord/meta.yaml:20]: should this be `build`, or is `host` correct?

Comment (ghutchis) [reply recipes/chemcoord/meta.yaml:20]: That part was from `grayskull` but host makes sense:
- pip is always required for host
- everything requires python >=3.6

**Review by bernt-matthias — APPROVED** [review]

**Comment by mcocdawc** [comment]
May I ask if there is anything that stops this PR from being merged.

The point is that I just released an updated version (2.1.0) of the library to PyPi.
It then makes also sense to update the conda-forge recipes.
Shall we still proceed with this slightly outdated version and then add another updated version, or shall we close this PR and reopen it for the new version?

@ghutchis FYI: the new version contains in particular the bugfixes for the changed pandas and numba APIs.

**Comment by bernt-matthias** [comment]
You can just commit an update to this branch. 

> May I ask if there is anything that stops this PR from being merged.

Not in my opinion. I'm not a conda forge member .. so I can't merge. 

You can only ping the team again or ask at https://app.element.io/#/room/#conda-forge:matrix.org if there is still no merge here I would switch to bioconda .. I had the same experience lately :(

**Comment by ghutchis** [comment]
@mcocdawc - we might as well update this recipe for the new version .. and then ping again.

**Comment by mcocdawc** [comment]
Thank you for both your answers. 

I see that the PR is from your (@ghutchis) private Fork, so I guess I have no write permissions there.
May I perhaps ask you to update the recipe?
The changes are:
* The version number is now 2.1.0
* The patch of the characters in the README is not required anymore.
* The commit hash of v2.1.0 is `c21cf5a7b5432c2cd6a8f0c5617d04c360776488`
* The numpy dependency is relaxed to `numpy>=1.0`. (From previously `numpy>=1.21`)

**Comment by ghutchis** [comment]
Should be all set.

@mcocdawc - for future reference, the sha256 needed is of the tar.gz on pipy, not the commit hash.

For now, I guess we're limiting to pandas < 2.0?

**Comment by ghutchis** [comment]
@conda-forge-admin, please ping @conda-forge/help-python

**Comment by mcocdawc** [comment]
Thank you very much!

**Pull request merged**